### PR TITLE
moved google font import out of css into js to make it optional

### PR DESCRIPTION
**Pull request by pburnsdata**
# Issues
#238 

Decided to remove font import from css

### CODE REVIEW

File: src/util/addFontCSS.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export default function addFontCSS() {
+    const context = this;
+
+    const fonts_url = '//fonts.googleapis.com/css?family=Open+Sans:400,300'
+
+    const styles = ['@import url('+ fonts_url +');'];
+
+    //Attach styles to DOM.
+    this.style = document.createElement('style');
```
Comment (pburnsdata) [inline src/util/addFontCSS.js]: might move this inside the if statement...

File: build/webcharts.js [hunk 2]
```diff
@@ -250,6 +250,30 @@
...
+
+        var request = new XMLHttpRequest();
+        request.open('HEAD', '//fonts.googleapis.com/css?family=Open+Sans:400,300');
+        request.send();
+
+        request.onreadystatechange = function() {
+            if (this.readyState == this.HEADERS_RECEIVED) {
+                console.log('gucci babies');
+                context.style.type = 'text/css';
+                context.style.innerHTML = styles.join('\n');
```
Comment (pburnsdata) [inline build/webcharts.js]: also probs dont need the whole array join thing

Comment (jwildfire) [reply build/webcharts.js]: Let's remove the stray console log before merge .

**Comment by jwildfire** [comment]
Looks like there is a [known issue](https://github.com/jsdom/jsdom/issues/1436) in jsdom that is causing the tests to fail here. Should be able to figure out a work around (something like [this](https://stackoverflow.com/questions/37059010/workarounds-for-jsdom-document-readystate-being-readonly)? )... or we could just add a setting instead.

**Comment by pburnsdata** [comment]
Yeah good point @jwildfire, that workaround looks like they're trying to set the state tho and looking through the tests... it looks like it's choking on XMLHttpRequest - might not be available to mocha since its in node?

**Comment by pburnsdata** [comment]
Decided to just remove the css font import - seemed like an unnecessary dependency. However, I'm not aware of how much of a difference the font makes. If either of you @jwildfire or @samussiah  think it's important we can find away to make this work, just might take some more thinking

**Pull request merged**